packages/requisitions/src/RequestRequisition/DetailView/Toolbar/ToolbarActions.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {
+  Autocomplete,
+  InputWithLabelRow,
+  useTranslation,
+  useConfirmationModal,
+} from '@openmsupply-client/common';
+import { useRequestFields, useIsRequestDisabled } from '../../api';
+
+const months = [1, 2, 3, 4, 5, 6];
+
+export const ToolbarActions = () => {
+  const t = useTranslation('replenishment');
+  const isDisabled = useIsRequestDisabled();
+  const { minMonthsOfStock, update } = useRequestFields('minMonthsOfStock');
+  const getConfirmation = useConfirmationModal({
```

Comment:
feel like I'm just throwing this everywhere at this point. I kind of like it and I think it makes the updates on the whole document quite nice, rather than sending a request and not getting feedback and waiting for the response to change a bunch of values